src/devices/cpu/z80/z80.lst

```diff
@@ -8817,15 +8403,14 @@ z80n:ed36	# add bc,**
 	BC += TDAT;
 
 z80n:ed8a	# push **
-	call arg16
-	using std::swap; swap(TDAT_H, TDAT_L);
-	call push
+	call arg
+	TDAT_H = TDAT_L;
+	call arg
+	call push TDAT
```

Comment:
This looks a little odd in the diff, but I’ll trust you on it.

Reply:
That's ok. arg16 swaps the values and here we had to swap them again. Just inlined without mirrored swaps